File: ytrue-bean/src/main/java/com/ytrue/bean/dataobject/system/SysJob.java
package com.ytrue.bean.dataobject.system;

import com.baomidou.mybatisplus.annotation.TableName;
import com.ytrue.bean.dataobject.BaseEntity;
import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

import java.io.Serial;

/**
 * @author ytrue
 * @description: SysJob
 * @date 2022/12/7 10:54
 */
@EqualsAndHashCode(callSuper = true)
@ToString(callSuper = true)
@Data
@TableName("sys_job")
public class SysJob extends BaseEntity {

    @Serial
    private static final long serialVersionUID = 8557945474976276338L;


    @Schema(description = "岗位名称")
    private String jobName;

    @Schema(description = "岗位排序")
    private Long jobSort;


    @Schema(description = "是否启用")
    private Boolean status;

}

File: ytrue-admin-server-csharp/Ytrue.Models/Entities/System/SysJob.cs
using SqlSugar;
using Ytrue.Models.Bases;

namespace Ytrue.Models.Entities.System;

/// <summary>
/// 系统岗位实体
/// </summary>
[SugarTable("sys_job")]
public class SysJob : BaseEntity
{
    /// <summary>
    /// 岗位名称
    /// </summary>
    [SugarColumn(ColumnName = "job_name")]
    public string JobName { get; set; } = null!;

    /// <summary>
    /// 岗位状态:0=禁用,1=启用
    /// </summary>
    [SugarColumn(ColumnName = "status")]
    public bool Status { get; set; }

    /// <summary>
    /// 排序
    /// </summary>
    [SugarColumn(ColumnName = "job_sort")]
    public int JobSort { get; set; }
}